test/core/rasterization/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+# Boost.GIL (Generic Image Library) - tests
+#
+# Copyright 2019 Miral Shah <miralshah2211@gmail.com>
```

Comment:
I'd replace Miral with yourself here, or add yourself.

I know this file may not reach the [certain threshold of originality](https://www.boost.org/users/license.html#FAQ) but it is not terribly essential, and it is part of the rasterization package authored by you.